setup.py

```diff
@@ -179,7 +179,7 @@ def get_proj_libraries():
     proj_libraries = ["proj"]
     if os.name == "nt" and (6, 0, 0) <= proj_version < (6, 3, 0):
         proj_libraries = [
-            "proj_{}_{}".format(proj_version[0], proj_version[1])
+            f"proj_{proj_version[0]}_{proj_version[1]}"
```

Comment:
You can't make this change to `setup.py`; it needs to remain Python 2 compatible.

Reply:
No it doesn't, the requires_python will prevent pip from downloading the sdist

Reply:
It does; read the comment at the top.